ops/pebble.py

```diff
@@ -62,7 +63,7 @@ def __init__(self, host, timeout=_not_provided, socket_path=None):
     def connect(self):
         """Override connect to use Unix socket (instead of TCP socket)."""
         if not hasattr(socket, 'AF_UNIX'):
-            raise NotImplementedError('Unix sockets not supported on {}'.format(sys.platform))
+            raise NotImplementedError('unix sockets not supported on {}'.format(sys.platform))
```

Comment:
Nit: I don't think this should be changed -- "Unix" is a proper noun so should be capitalized.

Reply:
fixed.